src/model/mod.rs

```diff
@@ -435,8 +481,8 @@ mod tests {
     #[test]
     fn test_same_pileup() {
         let variant = Variant::Deletion(3);
-        let tumor_all = AlleleFreq(0.0)..AlleleFreq(1.0);
-        let tumor_alt = AlleleFreq(0.0)..AlleleFreq(1.0);
+        let tumor_all = AFRange::inclusive( AlleleFreq(0.0)..AlleleFreq(1.0) );
+        let tumor_alt = AFRange::inclusive( AlleleFreq(0.0)..AlleleFreq(1.0) ); // TODO: should this be left_exclusive() instead?
```

Comment:
When reading this, I think the API can be even more improved. Having AFRange and AlleleFreq in the invocation is somehow redundant. Would be better to allow this:
```
AFRange::inclusive(0.0..1.0);
```